src/js/components/awardv2/visualizations/amounts/AggregatedAwardAmounts.jsx

```diff
@@ -23,28 +23,149 @@ const propTypes = {
     jumpToSection: PropTypes.func
 };
 
+const tooltipMap = {
+    obligated: "showObligatedTooltip",
+    current: "showCurrentTooltip",
+    potential: "showPotentialTooltip",
+    exceedsCurrent: "showExceedsCurrentTooltip",
+    exceedsPotential: "showExceedsPotentialTooltip"
+};
+
 export default class AggregatedAwardAmounts extends React.Component {
     constructor(props) {
         super(props);
+        this.state = {
+            showObligatedTooltip: false,
+            showCurrentTooltip: false,
+            showPotentialTooltip: false,
+            showExceedsCurrentTooltip: false,
+            showExceedsPotentialTooltip: false
+        };
 
         this.jumpToReferencedAwardsTable = this.jumpToReferencedAwardsTable.bind(this);
+        this.closeObligatedTooltip = this.closeTooltip.bind(this, "obligated");
+        this.showObligatedTooltip = this.showTooltip.bind(this, "obligated");
+        this.closeCurrentTooltip = this.closeTooltip.bind(this, "current");
+        this.showCurrentTooltip = this.showTooltip.bind(this, "current");
+        this.closePotentialTooltip = this.closeTooltip.bind(this, "potential");
+        this.showPotentialTooltip = this.showTooltip.bind(this, "potential");
+        this.closeExceedsCurrentTooltip = this.closeTooltip.bind(this, "exceedsCurrent");
+        this.showExceedsCurrentTooltip = this.showTooltip.bind(this, "exceedsCurrent");
+        this.closeExceedsPotentialTooltip = this.closeTooltip.bind(this, "exceedsPotential");
+        this.showExceedsPotentialTooltip = this.showTooltip.bind(this, "exceedsPotential");
+    }
+
+    getTooltipProps(chartType) {
+        switch (chartType) {
+            case "exceedsCurrent":
```

Comment:
Gotta be a better way to do this as well... probably by using a map of some kind?